src/components/EmailList/EmailList.jsx

```diff
@@ -0,0 +1,63 @@
+import EmailItem from "../EmailItem/EmailItem";
+import SearchBar from "../SearchBar/SearchBar";
+import EmailString from "../EmailString"
+import { useEffect, useState } from "react"
+
+function EmailList() {
+
+    const [emailString, setEmailString] = useState(false)
+    const emailItems = [];
+
+    function fetchJSON() {
+        
+        useEffect(() => {
+            fetch("https://email-client-api.dev.io-academy.uk/emails")
+            .then(response => response.json())
+            .then(data => {
+                console.log(data)
+                setEmailString(data)
+                return(
+                    emailString
+                )
+            })
+        }, [])
+
+    }
+
+    fetchJSON();
+
+    const [filteredEmailItems, setFilteredEmailItems] = useState(emailString);
```

Comment:
Generally we like to put all of the variable declarations, including state, at the top of the containing function. This is so that we know where to look when adding future code, to see if there is already state for the feature we want to add.

It also makes that state available for every function declared below it in the file - could be important!

Reply:
Moving this to the top of the page will break everything on the page

Reply:
ignore previous comment, this whole section was removed